lib/providers/conversations.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:chirp_nets/models/conversation.dart';
+import 'package:flutter/material.dart';
+import 'package:chirp_nets/utils/database.dart';
+
+class Conversations with ChangeNotifier {
+  Conversations() {
+    init();
+  }
+
+  Map<int, Conversation> _conversations = {};
+
+  Map<int, Conversation> get conversations {
+    return {..._conversations};
+  }
+
+  void init() async {
```

Comment:
What happens if they is not run immediately due to being async? could this cause problems on phones with less computation power, which may not get to run this?

Reply:
Since the value is an empty map (key:value pairs) if its isn't run immediately the page is empty until it loads

Reply:
Perhaps we can add an indicator that the conversation are still loading, as apposed to conversations not existing

Reply:
Theres a bunch of styling and quality of life stuff that still needs to be done, loading screen is on the list